src/storage/ScanVertexProcessor.cpp

```diff
@@ -0,0 +1,107 @@
+/* Copyright (c) 2019 vesoft inc. All rights reserved.
+ *
+ * This source code is licensed under Apache 2.0 License,
+ * attached with Common Clause Condition 1.0, found in the LICENSES directory.
+ */
+#include "storage/ScanVertexProcessor.h"
+#include "base/NebulaKeyUtils.h"
+#include <algorithm>
+#include <limits>
+#include "time/WallClock.h"
+#include "kvstore/RocksEngine.h"
+
+DECLARE_int32(max_scan_block_size);
+
+namespace nebula {
+namespace storage {
+
+void ScanVertexProcessor::process(const cpp2::ScanVertexRequest& req) {
+    auto spaceId = req.get_space_id();
+    auto partId = req.get_part_id();
+    std::string start;
+    std::string prefix = NebulaKeyUtils::prefix(partId);
+
+    if (req.get_cursor() == nullptr || req.get_cursor()->empty()) {
+        if (req.get_vertex_id() != nullptr) {
+            auto vId = *req.get_vertex_id();
+            start = NebulaKeyUtils::vertexPrefix(partId, vId);
+        } else {
+            start = NebulaKeyUtils::prefix(partId);
+        }
+    } else {
+        start = *req.get_cursor();
+    }
+
+    if (req.get_vertex_id() != nullptr) {
+        auto vId = *req.get_vertex_id();
+        prefix = NebulaKeyUtils::vertexPrefix(partId, vId);
+    } else {
+        prefix = NebulaKeyUtils::prefix(partId);
```

Comment:
the prefix default value is NebulaKeyUtils::prefix(partId); so here no need else

Reply:
Good suggestion